setup.py

```diff
@@ -96,6 +96,17 @@ def fetch_requirements(path):
     version_ge_1_5 = ['-DVERSION_GE_1_5']
 version_dependent_macros = version_ge_1_1 + version_ge_1_3 + version_ge_1_5
 
+import cpufeature
```

Comment:
Can we add `cpufeature` as a new dependency in this file? https://github.com/microsoft/DeepSpeed/blob/staging-zero-dual-v4/requirements/requirements.txt